examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt

```diff
@@ -0,0 +1,22 @@
+interface Greet { def sayHello(): Unit }
+interface Welcome { def sayHello(): Unit }
+
+def helloWorld() = try {
+  sayHello() 
+  //[info] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: There is an equally named effect operation sayHello of interface Greet. Use syntax `do sayHello()` to call it.
+  //     sayHello() 
+  //     ^^^^^^^^
+  // [info] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: There is an equally named effect operation sayHello of interface Welcome. Use syntax `do sayHello()` to call it.
+  //     sayHello() 
+  //     ^^^^^^^^
+  // [error] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: Cannot find a function named `sayHello`.
+  //     sayHello() 
+  //     ^^^^^^^^
```

Comment:
I wouldn't necessarily do this tbh, we have either:
1. the separate check files,
2. or the "write the error message in a comment"-style without needing a separate check file

for this exact purpose.

Doing both half-way could be somewhat confusing...